CMakeLists.txt

```diff
@@ -590,6 +603,7 @@ endif()
 
 if(SINGULARITY_BUILD_TESTS)
   include(CTest)
+  enable_testing()
```

Comment:
I don't know how tests ever ran without this. Did cmake get stricter at some point?